Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contest/ContestPage.tsx

```diff
@@ -20,7 +21,7 @@ const ContestPage = () => {
         (async () => {
             const contest = {
                 id: Number(contestId),
-                isOfficial: participationType === 'compete',
+                isOfficial: participationType === ContestParticipationType.Compete,
```

Comment:
`isOfficial` can, and should, be extracted into a `memo` variable outsite the `useEffect`.

This way, if `participationType` changes, this `useEffect` will be reexecuted.

Reply:
The `participationType` is added as a dependency to the `useEffect`. Isn't this enough to be reexecuted?